packages/graphback-codegen-resolvers/tests/snapshots/ApolloServiceResolverGeneratorTest.ts.md

```diff
@@ -54,8 +125,9 @@ Generated by [AVA](https://ava.li).
         {
           name: 'note',
           output: `import { validateRuntimeContext } from "@graphback/runtime";␊
+          import { Resolvers} from '../../generated-types␊
           ␊
-          export const noteResolvers = {␊
+          export const noteResolvers : Resolvers = {␊
```

Comment:
```
expected nospace before colon in variable-declaration (typedef-whitespace)tslint(1)
```

Reply:
WOW. Good catch. I was thinking that I have fixed that problem, but.. I have fixed it in generated file 🤦‍♂